tools/osbuilder/rootfs-builder/ubuntu/superprotocol/k8s.yaml

```
@@ -337,6 +337,14 @@ metadata:
   name: sp-argo-vm
   namespace: argocd
 spec:
+  ignoreDifferences:
+    - kind: Application
```

yamllint (1.35.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)